### PR TITLE
 scikit_learn issues

### PR DESCRIPTION
**Pull request by ndurumo254**

### CODE REVIEW

**Comment by ndurumo254** [comment]
#10401

**Comment by ndurumo254** [comment]
hi @YushaArif99 
hope this finds you well. kindly can you check my pr

File: ivy/functional/frontends/scikit-learn/Clustering/Functions.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    n_init="warn",
+    max_iter=300,
+    verbose=False,
+    tol=1e-4,
+    random_state=None,
+    copy_x=True,
+    algorithm="lloyd",
+    return_n_iter=False,
+):
+    return ivy.k_means(x)
```
Comment (hirwa-nshuti) [inline ivy/functional/frontends/scikit-learn/Clustering/Functions.py:16]: We currently don't have this function on the functional API🙂

You can implement it using compositions😊

File: ivy_tests/test_ivy/test_frontends/test_scikit_learn/test_Clustering/test_Functions.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+import ivy_tests.test_ivy.helpers as helpers
+from ivy_tests.test_ivy.helpers import assert_all_close
+from ivy_tests.test_ivy.helpers import handle_frontend_test
+
+
+@handle_frontend_test(
+    fn_tree="scikit-learn.k_means",
+    dtype_and_x=helpers.dtype_and_values(available_dtypes=helpers.get_dtypes("float")),
+)
+def scikit_learn_k_means(
```
Comment (hirwa-nshuti) [inline ivy_tests/test_ivy/test_frontends/test_scikit_learn/test_Clustering/test_Functions.py:11]: the testing function should have a name starting with `test_fn_name`

**Review by hirwa-nshuti — CHANGES_REQUESTED** [review]
Hey @ndurumo254 

Excellent job so far but generally many of the functions you implemented here are not on our functional API which means that many functions of this frontend will be implemented using the composition of other ivy functions🙂
Kindly go through the below comments and change the implementations accordingly, also update other functions which are not on the API as well😊

Thanks

**Pull request closed without merging**